server/codecity.js

```diff
@@ -148,7 +148,7 @@ CodeCity.parseJson = function(text) {
 CodeCity.allCheckpoints = function() {
   var files = fs.readdirSync(CodeCity.databaseDirectory);
   files = files.filter((file) => CodeCity.allCheckpoints.regexp_.test(file));
-  files.sort();
+  files.sort().reverse();
```

Comment:
Line 63: 
var checkpoint = CodeCity.allCheckpoints().pop();
Check if this needs to be [0] instead of pop.

Reply:
(It now is.)